Makefile

```diff
@@ -200,7 +200,7 @@ deploy: # Deploy/redeploy previously built REANA cluster.
 			sleep ${TIMECHECK}; \
 		fi;\
 	done && \
-	source ${PWD}/scripts/create-admin-user.sh ${TRUNC_INSTANCE_NAME} && \
+	source ${PWD}/scripts/create-admin-user.sh ${TRUNC_INSTANCE_NAME} admin@reana.io mysecretpassword && \
```

Comment:
I added these values here for compatibility but I guess Makefile will become obsolete soon.

Reply:
I would change `mysecretpassworde` for `123456` as it is the one we classicly use in these cases.